src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -121,8 +121,7 @@
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type TreeViewItem}">
-                            <Button Command="{Binding ClickedCommand}"
-                                    Focusable="False"
```

Comment:
Is there any way to say ScrollViewer not to scroll. Maybe `ScrollChanged` event can turn scrolling off?